bool return_pointcloud
bool return_rgb
bool return_depth
bool return_mask
